github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-postsubmits.yaml

```diff
@@ -1000,7 +1000,8 @@ postsubmits:
           - -c
           args:
           - |
-            bazel run //robots/cmd/kubevirt:kubevirt -- get presubmits --job-config-path-kubevirt-presubmits $(pwd)/github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml --github-token-path '' --output-file /tmp/presubmits.html
+            # we fetch presubmits for kubevirt/kubevirt branches main and release-0.59
+            bazel run //robots/cmd/kubevirt:kubevirt -- get presubmits --job-config-path-kubevirt-presubmits $(pwd)/github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml --job-config-path-kubevirt-presubmits $(pwd)/github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits-0.59.yaml --github-token-path '' --output-file /tmp/presubmits.html
```

Comment:
when we release 0.60, will we have to remember to update this to the 0.60 presubmits?

Reply:
Yes. I was thinking about automating this, but I'll leave that open for now...

Reply:
Something like

```
ls github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits* | sort -rV | head -1
```

Reply:
> Yes. I was thinking about automating this, but I'll leave that open for now...

What I mean by that is that I don't plan to do this inside this PR :wink: